lib/screenplay_web/controllers/config_controller.ex

```diff
@@ -79,4 +110,13 @@ defmodule ScreenplayWeb.ConfigController do
   defp place_id_has_screen?(place_id, app_id, _),
     do:
       raise("place_id_has_screen/2 not implemented for app_id: #{app_id}, place_id: #{place_id}")
+
+  # TODO: This is yucky but maybe ok?
+  # Need a function that can take an arbitrary screen config and find the place it's associated with.
+  # Finding this via the places_and_screens config would be expensive if we need t do it for a bunch of screen configs
+  defp screen_to_place_id(%Screen{app_params: %GlEink{footer: %{stop_id: stop_id}}}),
```

Comment:
I think this is fine, and is similar to what I made in the backend piece of this work ([here](https://github.com/mbta/screenplay/blob/9f50857a1d104a9bd821c3c1848f4e14fa6584e6/lib/screenplay/config/config.ex#L263)). Since most of our apps hold the stop IDs in different places, this seems like the natural way to do it.